test/config-luatest/upgrade_test.lua

```diff
@@ -101,6 +102,32 @@ g.test_upgrade = function()
             box.space._schema:replace{'version', 2, 11, 1}
         end)
 
+        local function alert_exists(alert)
+            local alerts = box.info.config.alerts
+            for _, a in pairs(alerts) do
+                if a.message == alert.message and a.type == alert.type then
+                    return true
+                end
+            end
+            return false
+        end
+
+        local err_msg = (
+            'The schema version %s is outdated, the latest ' ..
+            'version is %s. Please, consider using box.schema.upgrade().')
+            :format(mkversion(2, 11, 0), mkversion.get_latest())
+
+        -- Checks the alert for the outdated schema version exists.
+        t.assert(alert_exists({type = 'warn', message = err_msg}),
+                 'Outdated schema version alert exists')
+
+        box.schema.upgrade()
```

Comment:
Seems like it breaks the idea of the privileges upgrade test below. We want to verify that the privileges are granted on a non-lates schema.

There is `box.schema.upgrade()` at end of the test case. Let's place the alert check after it.

Hm. The alert check is already there :)

Reply:
Oh, I've left this extraneous `box.schema.upgrade()` here while resolving a merge conflict, thanks for pointing this out! Also I removed the redundant check for the alert not being present after schema was upgraded.

Reply:
I've reworked the test changes.